### PR TITLE
Feature/galaxy idempotency

### PR DESCRIPTION
**Pull request by ghost**
For motivation see commit message to 8948f40.

### CODE REVIEW

**Comment by Lujeni** [comment]
Hello,

This PR is hard to review and it's not the best for a fast integration imo.
- You change the `shebang` in your commit, maybe need a core dev validation before change this kind of stuff.
- You can `rebase` your branch also

> COMMIT: Editor automatically strips trailing whitespace
> 
> ``` bash
> $ git add -p
> ```

Rgd

File: bin/ansible-galaxy [hunk 1]
```diff
@@ -788,6 +789,17 @@ def execute_install(args, options, parser):
...
 
+        # if operations are not forced do not re-install an existing role
+        if not force:
+            try:
+                galaxy_install_info = get_galaxy_install_info(role.get("name"), options)
+                if galaxy_install_info.get("version") == role.get("version"):
+                    print "- %s already installed in version %s" % (role.get('name'), role.get('version'))
+                    roles_done.append(role)
+                    continue
+            except:
```
Comment (Lujeni) [inline bin/ansible-galaxy:800]: Maybe an explicite exception is better?

**Comment by jimi-c** [comment]
As the ansible-galaxy CLI stuff has been rewritten quite a bit since this issue was raised, can anyone confirm if they're still seeing it as a problem?

**Comment by bcoca** [comment]
since there was no response to the original review nor the latest feedback, I'm going to close this issue.

Feel free to reopen or create a new PR when ready to continue.

**Pull request closed without merging**